homeassistant/components/mill/climate.py

```diff
@@ -81,33 +93,33 @@ async def set_room_temp(service):
 class MillHeater(ClimateEntity):
     """Representation of a Mill Thermostat device."""
 
-    def __init__(self, heater, mill_data_connection):
+    def __init__(self, heater: Heater, mill_data_connection: Mill) -> None:
         """Initialize the thermostat."""
         self._heater = heater
         self._conn = mill_data_connection
 
     @property
-    def supported_features(self):
+    def supported_features(self) -> int:
         """Return the list of supported features."""
         return SUPPORT_FLAGS
 
     @property
-    def available(self):
+    def available(self) -> Any:
```

Comment:
```suggestion
    def available(self) -> bool:
```

Reply:
Would require the mill library to support typing?

Reply:
Yes. It will also need to export `py.typed` marker.